src/styles/blocks/catalog.scss

```
@@ -0,0 +1,40 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  justify-items: center;
+
+  gap: 48px;
+  padding: 0 30px 30px;
+  max-width: 1024px;
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
```

```suggestion
      grid-template-columns: repeat(2, 200px);
```

no need extra nesting here